test/scala_export/Dependency.scala

```diff
@@ -0,0 +1,7 @@
+package scalarules.test.scala_export
+
+class Dependency {
+    def getName(): String = {
+        "scala_export test"
+    }
+}
```

Comment:
This is such a small thing, but please add newlines to the end of these test `.scala` files.